internal/resources/resource_host.go

```diff
@@ -52,6 +53,23 @@ const (
 	allowedImageLength = 2
 )
 
+// Host State values.
+const (
+	hStateNew                 = "New"
+	hStateDeleting            = "Deleting"
+	hStateDeleted             = "Deleted"
+	hStateUpdatingConnections = "Updating Connections"
+	hStateImagingPrep         = "Imaging Prep"
+	hStateImaging             = "Imaging"
+	hStateConnecting          = "Connecting"
+	hStateBooting             = "Booting"
+	hStateReady               = "Ready"
+	hStateMaintenance         = "Maintenance"
+)
+
+// the untyped int 10, 30, 60.
+const ten, thirty, sixty = 10, 30, 60
```

Comment:
these would be better tenMinutes, twentyMinutes, sixtyMinutes = 10*time.Minutes etc...
then they are the a time.Duration type and can be used directly later on - also a lot more readable

Reply:
I'd prefer Mark's suggestion, or better yet, shortTimeout, mediumTimeout, longTimeout, or some appropriate name. I don't like declaring constants for the sake of constants like this.

Reply:
now I remember why I didn't do tenSeconds, etc. lint:
Error: internal/resources/resource_host.go:74:2: time-naming: var tenSeconds is of type time.Duration; don't use unit-specific suffix "Seconds" (revive).
will go with shortTimeout, mediumTimeout, longTimeout